cli/cmd/deploy.go

```diff
@@ -1,47 +1,63 @@
 package cmd
 
 import (
+	"fmt"
 	"os"
 
 	"github.com/pchico83/i2kit/cli/providers/aws"
+	"github.com/pchico83/i2kit/cli/schemas/environment"
 	"github.com/pchico83/i2kit/cli/schemas/service"
 	"github.com/spf13/cobra"
 )
 
 //Deploy deploys an i2kit service
 func Deploy() *cobra.Command {
-	var file string
-	var space string
-	var dryRun bool
+	var servicePath string
+	var environmentPath string
 	cmd := &cobra.Command{
 		Use:   "deploy",
 		Short: "Deploy an i2kit service",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if err := validateEnvironment(); err != nil {
+			reader, err := os.Open(servicePath)
+			if err != nil {
 				return err
 			}
-			if err := initManifest(file); err != nil {
+			if err = service.Validate(reader); err != nil {
```

Comment:
@rberrelleza I am providing (for now "empty") validation errors to be used by external backend